includes/class-amp-theme-support.php

```diff
@@ -178,6 +207,7 @@ public static function register_hooks() {
 		 */
 		add_action( 'template_redirect', array( __CLASS__, 'start_output_buffering' ), 0 );
 
+		// Commenting hooks.
```

Comment:
During the upcoming cleanup, I would suggest to move comments, widgets etc. in their on classes.